client/src/components/Navbar.js

```diff
@@ -36,23 +37,33 @@ const SearchBar = styled.input`
 export const Navbar = () => {
   return (
     <nav className="nav">
-      <Title>Project Metroid</Title>
+      <Title className="title">Project Metroid</Title>
+      <button className="mobile-search-toggle nav__mobile--link">
+        <i class="fas fa-search"></i>
+      </button>
+      <button className="mobile-user-profile nav__mobile--link">
+        <i class="fas fa-user-circle"></i>
+      </button>
+      <button className="mobile-hamburger-menu nav__mobile--link">
```

Comment:
Same as above.  We should name this button something like `button button--nav`.  The --nav is the modifier.  We have lots of buttons and alot of them well share some styles but we want to modify some of them such as the nav--button.